Documentation regarding user code removal/acceptance (#22507)
qmk/qmk_firmware: docs/feature_userspace.md

# Userspace: Sharing Code Between Keymaps

> !> Please note, userspace submissions to the upstream `qmk/qmk_firmware` repository are no longer being accepted. The userspace feature itself remains functional and can be configured locally.

Reviewer: ```suggestion
!> Please note, userspace submissions to the upstream `qmk/qmk_firmware` repository are no longer being accepted. The userspace feature itself remains functional and is intended to be utilised locally.
```
